README.md

```diff
@@ -1,69 +1,187 @@
-Fili Web Service [![Travis](https://img.shields.io/travis/yahoo/fili.svg?maxAge=2592000?style=plastic)](Travis)  [![Download](https://api.bintray.com/packages/yahoo/maven/fili/images/download.svg) ](https://bintray.com/yahoo/maven/fili/_latestVersion)
-===============================================================================================================
+Fili 
+====
 
+[![Download](https://api.bintray.com/packages/yahoo/maven/fili/images/download.svg) ](https://bintray.com/yahoo/maven/fili/_latestVersion) [![Gitter](https://img.shields.io/gitter/room/yahoo/fili.svg?maxAge=2592000)](https://gitter.im/yahoo/fili) [![Travis](https://img.shields.io/travis/yahoo/fili/master.svg?maxAge=2592000)](https://travis-ci.org/yahoo/fili/builds/) [![Codacy grade](https://img.shields.io/codacy/grade/91fa6c38f25d4ea0ae3569ee70a33e38.svg?maxAge=2592000)](https://www.codacy.com/app/Fili/fili/dashboard) [![Codacy coverage](https://img.shields.io/codacy/coverage/91fa6c38f25d4ea0ae3569ee70a33e38.svg?label=test coverage&maxAge=2592000)](https://www.codacy.com/app/Fili/fili/dashboard) [![VersionEye](https://img.shields.io/versioneye/d/user/projects/57eb1529bd6fa600512e4af1.svg?maxAge=2592000)](https://www.versioneye.com/user/projects/57eb1529bd6fa600512e4af1)
 
-Repository for the Fili web service.
+Fili is a Java-based framework that makes it easy to build and maintain RESTful web services for time series reporting 
+and analytics. Fili's HTTP GET-based [reporting API](docs/end-user-api.md) is clean and user-friendly, hiding the 
+complexities of [complicated metric and dimension definition](docs/configuring-metrics.md), data storage, and query 
+optimization from end-users. Designed with Big Data and scalability in mind, Fili has first-class support for
+[Druid](http://druid.io) as a back-end, but Fili's flexible pipeline-style architecture can handle nearly any back-end
+for data storage.
 
-## Monitoring and Operations
+Fili exposes the same core concepts that all time series reporting and analytics systems expose:
 
-There is [a list of Key Performance Indicators (KPIs)](kpis.md) for the web service. It doesn't go into detail about what each of the KPIs mean, but it gives a rough overview of why they matter.  
+- [Metrics](docs/end-user-api.md#metrics)
+- [Dimensions](docs/end-user-api.md#dimensions)
+- [Tables](docs/end-user-api.md#tables)
+- Time ([Reporting Time Grain](docs/end-user-api.md#time-grain) and [Interval](docs/end-user-api.md#interval))
 
+The biggest difference between Fili and those other systems, however, is that Fili _only_ exposes these concepts. With
+just these core concepts, end-users can ask a huge range of questions, extracting business value by focusing on finding 
+answers to the _what_ and _why_. Other reporting systems expose end-users to other concepts like Views, Partitions,
+and metric formulas, forcing users to worry too much about _how_ to answer their questions, but not Fili. 
```

Comment:
What about something like:

```markdown
On the other hand, other reporting systems expose end-users to other concepts like Views, Partitions, and metric formulars, forcing users to worry too much about _how_ to answer their questions.
```
Or replace `On the other hand` with `Meanwhile` or `In contrast` or something similar.